Sources/WebAuthn/Ceremonies/Registration/PublicKeyCredentialCreationOptions.swift

```diff
@@ -18,26 +18,28 @@ import Foundation
 public struct PublicKeyCredentialCreationOptions: Codable {
     public let challenge: EncodedBase64
     public let user: PublicKeyCredentialUserEntity
-    public let relyingParty: PublicKeyCredentialRpEntity
-    public let publicKeyCredentialParameters: [PublicKeyCredentialParameters]
+    // swiftlint:disable:next identifier_name
+    public let rp: PublicKeyCredentialRpEntity
+    public let pubKeyCredParams: [PublicKeyCredentialParameters]
     public let timeout: TimeInterval
+    public let attestation: AttestationConveyancePreference
 }
 
 // MARK: - Credential parameters
 
 public struct PublicKeyCredentialParameters: Equatable, Codable {
     public let type: String
-    public let algorithm: COSEAlgorithmIdentifier
+    public let alg: COSEAlgorithmIdentifier
```

Comment:
Regular users of this library shouldn't care about this. They'll just encode this and send it to the frontend/ client without inspecting what's in there.

Internally following the naming conventions of the WebAuthn specs makes things a lot easier during development. But I'm happy to change it back since I know the codebase, I'm just worrying about newbies.

Reply:
Ok fair enough